### DISCUSSION TITLE
Don't work "MantineProvider"

### DISCUSSION THREAD

**Discussion by SergeyBiryukov34** [mantinedev/mantine#282] [Feedback]
Hello) no base font applied on the page
![Screenshot_1](https://user-images.githubusercontent.com/59689428/136839462-1e7b7972-148d-46a1-aeb5-8c3ea0c7c8bd.png)

**Comment by rtivital** [comment]
Hi @SergeyBiryukov34 Mantine does not expose any global styles if you want styles applied to non Mantine components you will need to include GlobalStyles, see documentation here – https://mantine.dev/theming/create-styles/#globalstyles-component

**Reply by SergeyBiryukov34** [reply]
Thanks) I put the component 'GlobalStyles' inside 'MantineProvider' before the output 'App' but that didn't solve the problem...help me figure it out please

**Reply by rtivital** [reply]
That's odd GlobalStyles add font to body element ([reference](https://github.com/mantinedev/mantine/blob/master/src/mantine-styles/src/theme/GlobalStyles.tsx#L14)). Maybe you forgot to include Montserrat font?

**Reply by rtivital** [reply]
You can check if styles were applied by inspecting body element 
![image](https://user-images.githubusercontent.com/10353856/136844423-6c806a69-8599-4a4d-b713-4c5fe7ab60e5.png)

**Reply by SergeyBiryukov34** [reply]
Font included in index.html with google fonts..
![Screenshot_2](https://user-images.githubusercontent.com/59689428/136845032-b9bd0d96-2a9e-48ff-b7f9-2fc0bae412b0.png)

**Reply by rtivital** [reply]
You need to include separate font definition for headings:

```tsx
<MantineProvider theme={{ headings: { fontFamily: 'Greycliff CF, sans-serif' } }} />
```

You can find out more about other font properties here – https://mantine.dev/theming/extend-theme/

**Reply by SergeyBiryukov34** [reply]
![Screenshot_3](https://user-images.githubusercontent.com/59689428/136847676-275e31d9-0b00-45a6-a596-e67fbcb3a2a9.png)

**Reply by rtivital** [reply]
Please check that:
- you have styles on body element
- You actually render WithProvider component to the dom and not App component itself

**Reply by SergeyBiryukov34** [reply]
It was my mistake, I didn't include the WithProvider component in index.js

Can you please tell me how I can add "primary Color" to the theme of the "MantineProvider" component now? 
![Screenshot_1](https://user-images.githubusercontent.com/59689428/137088707-ca99376f-0b0f-4e50-a996-f29ee5930737.png)
![Screenshot_2](https://user-images.githubusercontent.com/59689428/137088723-add2cebe-494e-4278-b0a6-dbb59ebbeb84.png)

**Reply by rtivital** [reply]
primaryColor should match one of the keys from `theme.colors`, it cannot be a hex value

**Reply by dallanlee** [reply]
I had the same confusion with `theme.primaryColor` because the docs state that `primaryColor` should be `string` under the [theme object shape definition](https://mantine.dev/theming/mantine-provider/#theme-object)… Although I remember [reading in the docs](https://mantine.dev/theming/extend-theme/#primary-color) that "...you can change it to any color defined in theme.colors," I saw `'cyan'` in the example and assumed it was just using the CSS color keyword "`cyan`". Maybe a more detailed example could demonstrate that a custom color can be defined in `theme.colors` and used for the `theme.primaryColor`.

For example:

```js
const MY_COLORS = {
  brand: [
    '#E7E9EE',
    '#D2D3D7',
    '#9D9FA4',
    '#5c5f66',
    '#373A40',
    '#2C2E33',
    '#25262b',
    '#1A1B1E',
    '#141517',
    '#101113',
  ],
}

const mantineThemeOverrides = {
  …
  colors: MY_COLORS,
  …
  primaryColor: 'brand',
  …
}
```

**Reply by rtivital** [reply]
Yep, that seems reasonable, PR is welcome 👍

**Reply by dallanlee** [reply]
Okay. I'll work on a PR.
